bridgetown-core/lib/bridgetown-core/tags/find.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Bridgetown
+  module Tags
+    class Find < Liquid::Tag
+      include Bridgetown::Filters::ConditionHelpers
+      include Bridgetown::LiquidExtensions
+
+      SYNTAX = %r!^(.*?) (where|in) (.*?),(.*)$!.freeze
+
+      def initialize(tag_name, markup, tokens)
+        super
+        if markup.strip =~ SYNTAX
+          @new_var_name = Regexp.last_match(1).strip
+          @single_or_group = Regexp.last_match(2)
+          @arr_name = Regexp.last_match(3).strip
+          @conditions = Regexp.last_match(4).strip
+        else
+          raise SyntaxError, <<~MSG
+            Syntax Error in tag 'find' while parsing the following markup:
+
+            #{markup}
+
+            Valid syntax: find <varname> where|in <array>, <condition(s)>
+          MSG
+        end
+      end
+
+      def render(context)
+        @group = lookup_variable(context, @arr_name)
+        return "" unless @group.respond_to?(:select)
+
+        @group = @group.values if @group.is_a?(Hash)
+
+        expression = @conditions.split(",").map do |condition|
```

Comment:
Hmm, I think I'm being naive here…if somebody has a comma inside of a string value, this is going to mess this up. I probably need to do a more intelligent syntax scan. Besides, I want to implement "or" logic at some point as well which will need further parsing smarts.